src/wallet/coinselection.h

```diff
@@ -109,6 +109,10 @@ class COutput
 
     bool m_from_me;
 
+    CAmount effective_value;
```

Comment:
864b4edf130afe2b77142ca343022b3de6d3fec5: did you mean `m_ effective_value`?

Reply:
Done